include/extractor/extraction_way.hpp

```diff
@@ -106,6 +108,8 @@ struct ExtractionWay
     bool roundabout;
     bool circular;
     bool is_startpoint;
+    bool backward_local_access_only;
```

Comment:
Using the term `restricted` instead of `local_access_only` might be more in line with our usual vocabulary for this. My suggestion would be `{backward,forward}_restricted`